CHANGELOG.md

```diff
@@ -12,7 +12,9 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ##### Bug Fixes
 
-* None.  
+* Filter out specs that are used by app specs only from linking into user
```

Comment:
nit this is a bit worded weirdly.

`Do not link specs into user targets that are only used by app specs`